keyboards/milk_truck/readme.md

```diff
@@ -0,0 +1,19 @@
+# Milk Truck
+
+![Milk Truck](https://imgur.com/NUmiTDS.jpg)
+
+A dairy-filled, left-handed, numpad-having, extended-minivan, forty-percent keyboard.
+
+* Keyboard Maintainer: [Ryan Boone](https://github.com/rjboone) (Drmmr#0004)
+* Hardware Supported: Milk Truck
+* Hardware Availability: Group buy from [P3D Store](https://p3dstore.com)
+
+Make example for this keyboard (after setting up your build environment):
```

Comment:
Could you add instructions on how to enter the bootloader, to flash new firmware? 

Eg: 
```markup
## Bootloader

Enter the bootloader in 3 ways:

* **Bootmagic reset**: Hold down the key at (0,0) in the matrix (usually the top left key or Escape) and plug in the keyboard
* **Physical reset button**: Briefly press the button on the back of the PCB - some may have pads you must short instead
* **Keycode in layout**: Press the key mapped to `RESET` if it is available
```